tests/compiler/test_database.py

```diff
@@ -1,9 +1,10 @@
+from operator import sub
```

Comment:
This appears unintended. Your IDE seem to auto import stuff?

Reply:
Thanks for reviewing. Yes, I will check how to turn it off.